app/controllers/user.rb

```diff
@@ -0,0 +1,53 @@
+
+
+## member routes
+
+
+
+## admin routes
+
+get '/users' do
+	@users = User.where(access_level: "member")
+	erb :'users/users_index'
+end
+
+get '/users/new' do
+	@errors = session.delete(:errors)
+	erb :'users/users_new'
+end
+
+get '/users/:user_id' do |id|
+	@user = User.find(id)
+	if @user.access_level == "member" 
+		erb :'users/users_show'
+	else
+		close_current_batch
+		@users = User.where(access_level: "member")
+		erb :'users/users_admin_show'
+	end
+end
+
+
+post '/users' do
+  add_member params[:new_user]
```

Comment:
You should be checking to see if the user creation succeeded or failed. Redirect if it works, re-render the new form with errors if it fails.